CoreTesting/src/au/gov/asd/tac/constellation/testing/construction/ImageGraphBuilderPlugin.java

```diff
@@ -81,6 +85,9 @@ public class ImageGraphBuilderPlugin extends SimpleEditPlugin {
     private static final Logger LOGGER = Logger.getLogger(ImageGraphBuilderPlugin.class.getName());
 
     public static final String IMAGE_FILE_PARAMETER_ID = PluginParameter.buildId(ImageGraphBuilderPlugin.class, "image_file");
+    public static final String ADD_RIGHT_PARAMETER_ID = PluginParameter.buildId(ImageGraphBuilderPlugin.class, "add_right");
+    public static final String ADD_LAYERS_PARAMETER_ID = PluginParameter.buildId(ImageGraphBuilderPlugin.class, "add_layers");
+    public static final String LAYER_START_PARAMETER_ID = PluginParameter.buildId(ImageGraphBuilderPlugin.class, "layer_start");
```

Comment:
This doesn't appear to be used anywhere.

Reply:
Removed LAYER_START_PARAMETER_ID